packages/fxa-auth-server/lib/senders/emails/templates/verifySecondary/index.mjml

```diff
@@ -0,0 +1,30 @@
+<%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
+
+<mj-include path="./css/global.css" type="css" css-inline="inline" />
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-header">
+      <span data-l10n-id="verifySecondary-title">Verify secondary email</span>
+    </mj-text>
+
+    <mj-text css-class="text-body">
+      <span data-l10n-id="verifySecondary-explainer" data-l10n-args='<%= JSON.stringify({email}) %>'>A request to use <%= email %> as a secondary email address has been made from the following Firefox Account:</span>
+    </mj-text>
+  </mj-column>
+</mj-section>
+
+<%- include('/partials/location/index.mjml') %>
+<%- include('/partials/button/index.mjml') %>
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-body-top-padded">
+      <span data-l10n-id="verifySecondary-post-verification">Once verified, this address will begin receiving security notifications and confirmations.</span>
+    </mj-text>
+  </mj-column>
+</mj-section>
+
+<%- include('/partials/automatedEmailNoAction/index.mjml') %>
```

Comment:
It looks like in the old template, there's much less space between the button and footer than what we've got in the new template. My original comment was "yay for consistency!" but as you click through the templates, we actually seem to be slightly inconsistent with the spacing here -  maybe we want a `margin-top` on `text-footer-email` - there's a lot of space in all of the `verify` templates, `cadReminder` templates, `passwordChanged`/`passwordReset`, and `postVerify`:

![image](https://user-images.githubusercontent.com/13018240/132391990-719b013a-f78b-44f4-91a1-47114ddb5dbf.png)


But then in some of our other templates, like `lowRecoveryKeys`, `newDeviceLogin`, `unblockCode`, and maybe one or two others that automated text doesn't have that spacing:

![image](https://user-images.githubusercontent.com/13018240/132392294-cc409fdf-17ea-4b27-922a-4ca9e5fe309d.png)

Reply:
@LZoog I did my best to make this a little more consistent, though I think this could use a task for a proper cleanup